common/include/kmx_file.h

```diff
@@ -25,7 +25,7 @@ typedef KMX_WORD  __attribute__((aligned(1))) KMX_WORD_unaligned;
 #define KMX_WORD_unaligned  KMX_WORD
 #endif
 
-#ifdef KMN_KBP
+#ifdef KMN_CORE
```

Comment:
Everywhere else the prefix is KM_CORE, do we want to use the same, and some thing after to make it clear it's a preprocessor control, e.g. KM_CORE_LIBRARY?